### DISCUSSION TITLE
Is it safe to use the library in production?

### DISCUSSION THREAD

**Discussion by Minh8181-blz** [Adyen/adyen-flutter#148] [Q&A] [answered]
As I see the library is still in alpha version. Currently we are building Flutter app that integrates with Adyen with this library. I want to know if it's safe to use it in production and will there be breaking changes in the future?

**Comment by camil-adyen** [comment, marked as answer]
Hi @Minh8181-blz, 

Thank you for reaching out.

We consider our Flutter SDK safe and ready for production. The reason why it's labeled as alpha is because we do not yet support all features we want the stable v1 to include. If you're looking to implement our Drop-in or a standalone Card component, you can definitely use our library in production. Our goal is to release an official v1 by the end of Q2 '24. There can still be breaking changes between now and the official v1 release. However, we aim to make the transition from alpha to beta to v1 as seamless as possible.
